index.html

```diff
@@ -222,12 +220,36 @@ <h3>Ruby</h3>
                     </li>
                 </ul>
             </div>
-           
+
             <button type="button">See Project </button>
-           
+
         </div>
 
+
     </section>
+    <footer class="f-contact">
+        <h3>CONTACT US</h3>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that
+            needs
+            coding. I’d love to help with it</p>
+        
+        <form action="https://formspree.io/f/moqrdewo" method="post">
+            <input type="text" name ="full name" placeholder="Text input" maxlength="30" required>
+            <input type="email" name="email" placeholder="khanetmoi@gmail.com" required>
```

Comment:
[OPTIONAL] Great job on choosing the correct `input types` :+1: following the [chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). However you may consider giving your placeholder text a bit more descriptive content. 

_hint: instead of displaying just `Text input` you can consider something like `Full name`._

_note: what you have done is enough to go further, it is just a friendly suggestion :relaxed: since giving correct associated placeholder text to your input fields ensures that you will get exact desired data from the user, especially it is important when we are not using `<label/>`_